src/plugins/content_management/demo/todo/todos.tsx

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import React from 'react';
+import { EuiButtonIcon, EuiCheckbox, EuiFieldText, EuiButtonGroup, EuiSpacer } from '@elastic/eui';
+
+import {
+  useSearchContentQuery,
+  useCreateContentMutation,
+  useDeleteContentMutation,
+  useUpdateContentMutation,
+  // eslint-disable-next-line @kbn/imports/no_boundary_crossing
+} from '../../public/content_client';
+import type { Todo } from './todos_client';
+import { SearchIn, SearchOut, CreateIn, DeleteIn, UpdateIn } from '../../common';
+
+export const Todos = () => {
+  const [filterIdSelected, setFilterIdSelected] = React.useState('all');
+
+  const { data, isLoading, isError, error } = useSearchContentQuery<SearchIn, SearchOut<Todo>>({
```

Comment:
For each of these hooks I had to manually define and specify `In` and `Out` which isn't convenient.